Dockerfile

```diff
@@ -55,10 +40,10 @@ RUN wget https://github.com/processone/iconv/archive/${ICONV_VERSION}.tar.gz \
 WORKDIR /tmp/iconv/iconv-${ICONV_VERSION}
 RUN rebar3 compile
```

Comment:
How does this work if you've commented out the rebar3 installation above ? Is it part of `elixir:1.8-otp-22-alpine` ?

Reply:
Yup, that exists on the base image.